resolvers.js

```diff
@@ -168,6 +190,33 @@ const resolvers = {
         }
       );
     },
+
+    updateDatabases: async (parent, { databases }, context) => {
+      assertAuthenticated(context);
+
+      const email = context.getUser().email;
+      const userDocument = await User.findOne({ email: email });
+
+      userDocument.databases = databases;
+
+      userDocument.save();
+
+      return userDocument;
+    },
```

Comment:
what is this doing? this is very dangerous... do whatever operation it is on the backend instead